css_tasks/task11/scss/_clients.scss

```diff
@@ -0,0 +1,99 @@
+@import "variables";
+@import "mixins";
+.clients {
+  position: relative;
+  display: flex;
```

Comment:
no reason for flex and flex column here, it is default how browser create a flow of element, this code have no sense

Reply:
Deleted